src/sentry/static/sentry/app/actionCreators/globalSelection.jsx

```diff
@@ -0,0 +1,25 @@
+/*eslint no-use-before-define: ["error", { "functions": false }]*/
+
+import {isInteger} from 'lodash';
+import GlobalSelectionActions from 'app/actions/globalSelectionActions';
+import sdk from 'app/utils/sdk';
+
+/**
+ * Updates global project selection
+ *
+ * @param {Number[]} projects List of project ids
+ */
+export function updateProjects(projects) {
+  if (!isProjectsValid(projects)) {
+    sdk.captureException('Invalid projects selected', {
```

Comment:
The first arg should be an `Error` instance
```suggestion
    sdk.captureException(new Error('Invalid projects selected'), {
```